decidim-accountability/app/commands/decidim/accountability/admin/create_result.rb

```diff
@@ -76,6 +77,20 @@ def link_projects
         def link_meetings
           result.link_resources(meetings, "meetings_through_proposals")
         end
+
+        def notify_proposal_followers
+          proposals.includes(:author).each do |proposal|
+            Decidim::EventsManager.publish(
+              event: "decidim.events.accountability.proposal_linked",
+              event_class: Decidim::Accountability::ProposalLinkedEvent,
+              resource: result,
+              recipient_ids: Array(proposal.author.id) + proposal.followers.pluck(:id),
```

Comment:
`[proposal.author_id]`?

Reply:
It should be proposal.decidim_author_id and I didn't want this code to know about the internals of how to get an id, although this would fail with an authorless proposal. I'll update it